test/Interop/Cxx/class/address-only.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %empty-directory(%t)
```

Comment:
Please rename it to `type-classification-non-trivial.swift` to be in line with other tests.